src/2025h2/FLS-up-to-date-capabilities.md

```diff
@@ -20,24 +20,32 @@ The target audience here will be those that have a vested interest in Rust speci
 
 Keeping up with documentation is not always the most glamorous role for any project. And folks like @ehuss and others are doing an outstanding job at trying to keep the Reference current with actual code behavior. This is a hard job just for one document like the Reference. It will be even more difficult adding another core document like the FLS. But that doesn't necessarily mean we can't keep the FLS as up to date as is needed by its users.
 
+### Solicitation of stakeholder involvement
+
+The FLS is an enabler document for safety-qualifying versions of the Rust compiler. The safety-critical community of Rust users actively and passively benefit from the FLS.
```

Comment:
```suggestion
The FLS is an enabler document for safety qualifying versions of the Rust compiler. The safety-critical community of Rust users actively and passively benefits from the FLS.
```

I think you mean here for "safety" to be an adverb qualifying the verb "qualifying", in which case I wouldn't put a dash here.  If instead you mean for "safety-qualifying" to be an adjective qualifying "versions", then the dash would be justifiable, but in that case I'd probably say "safety qualified versions" instead.